frontend/src/lib/components/Dropdown/dropdown.tsx

```diff
@@ -182,36 +217,36 @@ export function Dropdown<TValue = string>(props: DropdownProps<TValue>) {
     }, []);
 
     React.useEffect(
-        function handleOptionsChangeEffect() {
-            function handleMouseDown(event: MouseEvent) {
+        function setupInputBlurListener() {
+            // ! The ref is *actually* the div wrapping the input, so normal "blur" is not available
```

Comment:
There is an `inputRef` prop that is referencing the input element. You can add a ref to that to gain control over blur etc. However, I double-checked and there was something messed up with the references - I created a PR to fix the issues: https://github.com/equinor/webviz/pull/865